source/gameengine/Ketsji/KX_NavMeshObject.cpp

```diff
@@ -810,6 +817,19 @@ bool KX_NavMeshObject::BuildNavMesh()
     if (vertices) {
       delete[] vertices;
     }
+    if (dvertices) {
```

Comment:
There is another delete [ ] dvertices more down. The same occurs in the other checks.

Reply:
Thanks! I'll fix that.